src/main/java/seedu/address/logic/LogicManager.java

```diff
@@ -78,4 +80,36 @@ public GuiSettings getGuiSettings() {
     public void setGuiSettings(GuiSettings guiSettings) {
         model.setGuiSettings(guiSettings);
     }
+
+    @Override
+    public boolean saveAddressBookToCsv(Path csvFilePath) {
+        boolean result;
+        try {
+            storage.saveAddressBookToCsv(model.getAddressBook(), csvFilePath);
+            result = true;
+        } catch (IOException ioe) {
+            // throw new CommandException(FILE_OPS_ERROR_MESSAGE + ioe, ioe);  // TODO
+            result = false;
+        }
+        return result;
+    }
+
+    @Override
+    public boolean loadAddressBookFromCsv(Path csvFilePath) {
+        boolean result;
+        try {
+            Optional<ReadOnlyAddressBook> ab = storage.readAddressBookFromCsvFile(csvFilePath);
+            if (ab.isPresent()) {
+                model.setAddressBook(ab.get());
+                result = true;
+            } else {
+                System.out.println("Error");
```

Comment:
Is this line needed? (Line 106)

Reply:
Oh I did this to remind myself to add changes, but must have forgotten about this afterwards, will change this next time!

Reply:
Wait idk what happened but I ended up changing this code in the end but it's not reflected here